contrib/cmd/recvtty/recvtty.go

```diff
@@ -61,28 +61,35 @@ terminals:
 )
 
 func bail(err error) {
-	fmt.Fprintf(os.Stderr, "[recvtty] fatal error: %v\n", err)
+	_, _ = fmt.Fprintf(os.Stderr, "[recvtty] fatal error: %v\n", err)
 	os.Exit(1)
 }
 
-func handleSingle(path string, noStdin bool) error {
+func handleSingle(path string, noStdin bool) (err error) {
 	// Open a socket.
 	ln, err := net.Listen("unix", path)
 	if err != nil {
 		return err
 	}
-	defer ln.Close()
+	defer func() {
+		err = ln.Close()
```

Comment:
So, what would happen if `net.Listen` fails? Please be careful.

Reply:
I was wrong here (except that you rewriting the error).

Reply:
No need to metion the defer part will run after func returns. so if any error happens in closing, the `retErr` will be refilled. the question is: is it important to you to handle this kind of errors in the caller function?

Reply:
There is also another problem: if the func returns because of another error, and at the same time, the close also fails, the former error will be lost! while the closing error is bound (in my opinion) I believe the original error is more significant ... tell me what to do?

Reply:
I decided to ignore closing error...